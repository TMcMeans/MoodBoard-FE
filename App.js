import React from 'react';
import { StyleSheet, View } from 'react-native';
import Journal from './src/containers/Journal/Journal';

export default class App extends React.Component {
  render() {
    return (
      <View style={styles.container}>
        <Journal />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
