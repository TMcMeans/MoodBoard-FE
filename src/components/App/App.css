/*COMPONENT STYLES*/
.App {
  background-color: #2A2A2A;
  font-family: 'Raleway', sans-serif;
  height: 100vh;
  margin: 0;
  text-align: center;
}

/*GLOBAL BUTTON STYLES*/
.standard-btn {
  border: none;
  border-radius: 5px;
  box-shadow: 0px 1px 4px -2px #333;
  color: #ffffff;
  cursor: pointer;
  font-family: 'Raleway', sans-serif;
  font-weight: bold;
  position: relative;
  text-shadow: 0px -1px #333;
}

.standard-btn:after{
  content: '';
  position: absolute;
  top: 2px;
  left: 2px;
  width: calc(100% - 4px);
  height: 50%;
  background: linear-gradient(rgba(255,255,255,0.6), rgba(255,255,255,0.2));
}

/*BUTTON COLORS*/
.light {
  background-color: #daa520;
  background-image: linear-gradient(#c4941c, #e5c062);
}

.dark {
  background-color: #2a2a2a;
  background-image: linear-gradient(#2a2a2a, #696969)
}

.rainbow {

}

/*404 PAGE*/
#four-oh-four{
  color: #ffffff;
  padding: 5em;
}