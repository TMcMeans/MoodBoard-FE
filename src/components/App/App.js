import React, { Component } from 'react';
import { Switch, Route, withRouter, Link } from 'react-router-dom';
import Affirmations from '../../containers/Affirmations/Affirmations';
import CalendarView from '../../containers/CalendarView/CalendarView';
import DailyView from '../../containers/DailyView/DailyView';
import Home from '../../containers/Home/Home';
import Journal from '../../containers/Journal/Journal';
import TodayMood from '../../containers/TodayMood/TodayMood';
import UserForm from '../UserForm/UserForm';
import WriteAffirmation from '../../containers/WriteAffirmation/WriteAffirmation';
import Info from '../Info/Info.js';


import './App.css';

export class App extends Component {
  render() {
    return (
      <div className="App">
        <Switch>
          <Route exact path="/" render={() => <UserForm type="login" />} />
          <Route exact path="/signup" render={() => <UserForm type="signup" />} />
          <Route exact path="/home" render={() => <Home />} />
          <Route exact path="/calendarview" render={() => <CalendarView />} />
          <Route exact path="/dailyview" render={() => <DailyView />} />
          <Route exact path="/journal" render={() => <Journal />} />
          <Route exact path="/todaymood" render={() => <TodayMood />} />
          <Route exact path="/affirmations" render={() => <Affirmations />} />
          <Route exact path="/writeaffirmation" render={() => <WriteAffirmation />} />
          <Route exact path="/info" render={() => <Info />} />
          <Route path='' render={() =>
            (<div id="four-oh-four">
              <h1>Oops! We couldn't find that page.</h1>
              <Link to='/home'><button className="standard-btn">Home</button></Link>
              </div>)
            } />
        </Switch>
      </div>
    );
  }
}

export default withRouter(App);
