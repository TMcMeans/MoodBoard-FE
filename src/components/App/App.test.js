import React from 'react';
import { shallow } from 'enzyme';
import { App } from './App';

describe('App', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });
  it('should render a snapshot with correct data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
