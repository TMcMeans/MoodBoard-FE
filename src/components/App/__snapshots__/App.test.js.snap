// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render a snapshot with correct data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Switch>
        <Route
          exact={true}
          path="/"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/signup"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/home"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/calendarview"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/dailyview"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/journal"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/todaymood"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/affirmations"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/writeaffirmation"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/info"
          render={[Function]}
        />
        <Route
          path=""
          render={[Function]}
        />
      </Switch>,
      "className": "App",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Route
            exact={true}
            path="/"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/signup"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/home"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/calendarview"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/dailyview"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/journal"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/todaymood"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/affirmations"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/writeaffirmation"
            render={[Function]}
          />,
          <Route
            exact={true}
            path="/info"
            render={[Function]}
          />,
          <Route
            path=""
            render={[Function]}
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/signup",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/home",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/calendarview",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/dailyview",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/journal",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/todaymood",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/affirmations",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/writeaffirmation",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "exact": true,
            "path": "/info",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "",
            "render": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Switch>
          <Route
            exact={true}
            path="/"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/signup"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/home"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/calendarview"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/dailyview"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/journal"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/todaymood"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/affirmations"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/writeaffirmation"
            render={[Function]}
          />
          <Route
            exact={true}
            path="/info"
            render={[Function]}
          />
          <Route
            path=""
            render={[Function]}
          />
        </Switch>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Route
              exact={true}
              path="/"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/signup"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/home"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/calendarview"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/dailyview"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/journal"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/todaymood"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/affirmations"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/writeaffirmation"
              render={[Function]}
            />,
            <Route
              exact={true}
              path="/info"
              render={[Function]}
            />,
            <Route
              path=""
              render={[Function]}
            />,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/signup",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/home",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/calendarview",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/dailyview",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/journal",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/todaymood",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/affirmations",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/writeaffirmation",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "exact": true,
              "path": "/info",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "path": "",
              "render": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
